packages/shared/index.js

```diff
@@ -110,7 +110,6 @@ export {
 /**
  * Non-UI components (components without render output):
  */
-export { default as AppStateChange } from './src/AppStateChange';
```

Comment:
That file would have been relevant in a full-on RN app, right?

Reply:
It could be relevant in a brownfield app as well, but we created this long time ago, we had a use-case for it. We no longer use it, so we don't need dead code.

Reply:
Absolutely